docker/env/Dockerfile

```diff
@@ -1,15 +1,25 @@
-FROM centos:7
+FROM python:3.8
 
-ADD install-prereqs.sh install-prereqs.sh
-RUN ./install-prereqs.sh docker
-RUN yum update -y \
-    && yum install -y https://repo.ius.io/ius-release-el7.rpm \
-    && yum install -y python36u python36u-libs python36u-devel python36u-pip
+RUN DEBIAN_FRONTEND=noninteractive apt update && \
+    apt install -y git openssl iproute2 sudo apt-transport-https \
+    ca-certificates curl gnupg2 software-properties-common rsync
+
+# Install Docker cli only
+RUN curl -fsSL https://download.docker.com/linux/debian/gpg | sudo apt-key add - && \
+    DEBIAN_FRONTEND=noninteractive add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/debian $(lsb_release -cs) stable" && \
+    DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y docker-ce-cli  && \
+    sudo groupadd docker && \
+    sudo usermod -aG docker `whoami`
```

Comment:
I think you can replace `whoami` with `$USER`

Reply:
Tried it, it doesn't work. you need to define this environment variable for it to work.

maybe it's not needed at all..., since in hydra we pass all the group ids into the docker commandline

Reply:
Yes, you're right. I was mistakenly ensured that it should work.